arm-graphrbac/1.6-internal/swagger/graphrbac.json

```diff
@@ -1,225 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "title": "GraphRbacManagementClient",
-    "version": "1.6-internal",
```

Comment:
And then remove CompositeSwagger and refer 1.6 swagger in the scripts to generate sdk.